autokeras/blocks/basic.py

```diff
@@ -154,6 +154,54 @@ def build(self, hp, inputs=None):
         return output_node
 
 
+class AttentionBlock(block_module.Block):
+    """A Attention Block.
+
+    # Arguments
+        return_sequences: Boolean. Whether to return the last output in the
+            output sequence, or the full sequence. Defaults to False.
+    """
+
+    def __init__(self,
+                 return_sequences: bool = False,
+                 **kwargs):
+        super().__init__(**kwargs)
+        self.return_sequences = return_sequences
```

Comment:
I don't think we need this. If it is False, there is no need to use this attention block.